src/main/java/beans/menus/Style.java

```diff
@@ -18,7 +18,7 @@ public Style(DetailField detail){
             try {
                 setFontSize(Integer.parseInt(detail.getFontSize()));
             } catch (NumberFormatException nfe) {
-                nfe.printStackTrace();
+                //nfe.printStackTrace();
```

Comment:
something should be handled here or logging of the stack trace. maybe set a default fontsize?